_locales/en/messages.json

```diff
@@ -1395,6 +1395,9 @@
 	"hub_home_header_info": {
 		"message": "New users are opted in to participating in Human Web and Ghostery Rewards. You may change these settings in custom setup or in the extension. "
 	},
+	"hub_home_header_info_cliqz": {
+		"message": "New users are opted in to participating in Human Web. You may change these settings in custom setup or in the extension. "
+	},
```

Comment:
Since we only mention one setting for Cliqz, it should be 'this setting' in the second sentence here.

Reply:
Gotcha, fixed.